.github/workflows/test.yml

```diff
@@ -29,6 +29,7 @@ on:
 
 env:
   LC_ALL: en_US.UTF-8
+  REGISTRY_AUTH_FILE: /run/containers/1001/auth.json
```

Comment:
Is this going to work? The func test has `REGISTRY_AUTH_FILE=$HOME/auth.json`

Reply:
no need for this anymore. so I will be removing it

Reply:
removed